plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_uk.properties

```diff
@@ -0,0 +1,93 @@
+
+command.org.jkiss.dbeaver.erd.focus.diagram.name = ERD: \u0424\u043E\u043A\u0443\u0441 \u043D\u0430 \u0434\u0456\u0430\u0433\u0440\u0430\u043C\u0456
+command.org.jkiss.dbeaver.erd.focus.palette.name = ERD: \u0424\u043E\u043A\u0443\u0441 \u043D\u0430 \u043F\u0430\u043B\u0456\u0442\u0440\u0456
+command.org.jkiss.dbeaver.erd.focus.outline.name = ERD: \u0424\u043E\u043A\u0443\u0441 \u043D\u0430 \u043A\u043E\u043D\u0442\u0443\u0440\u0456
+command.org.jkiss.dbeaver.erd.focus.parameters.name = ERD: \u0424\u043E\u043A\u0443\u0441 \u043D\u0430 \u043F\u0430\u0440\u0430\u043C\u0435\u0442\u0440\u0430\u0445
+
+command.org.jkiss.dbeaver.erd.focus.diagram.description = \u0417\u043C\u0456\u043D\u0430 \u0444\u043E\u043A\u0443\u0441\u0443 \u043D\u0430 \u0434\u0456\u0430\u0433\u0440\u0430\u043C\u0456
+command.org.jkiss.dbeaver.erd.focus.palette.description = \u0417\u043C\u0456\u043D\u0430 \u0444\u043E\u043A\u0443\u0441\u0443 \u043D\u0430 \u043F\u0430\u043B\u0456\u0442\u0440\u0456
+command.org.jkiss.dbeaver.erd.focus.outline.description = \u0417\u043C\u0456\u043D\u0430 \u0444\u043E\u043A\u0443\u0441\u0443 \u043D\u0430 \u043A\u043E\u043D\u0442\u0443\u0440\u0456
+command.org.jkiss.dbeaver.erd.focus.parameters.description = \u0417\u043C\u0456\u043D\u0430 \u0444\u043E\u043A\u0443\u0441\u0443 \u043D\u0430 \u043F\u0430\u0440\u0430\u043C\u0435\u0442\u0440\u0430\u0445
+
+
+themeElementCategory.org.jkiss.dbeaver.ui.presentation.erd.label = ER \u0414\u0456\u0430\u0433\u0440\u0430\u043C\u0438
+themeElementCategory.org.jkiss.dbeaver.ui.presentation.erd.description = ER \u0414\u0456\u0430\u0433\u0440\u0430\u043C\u0438
+
+fontDefinition.org.jkiss.dbeaver.erd.diagram.font.label = \u0428\u0440\u0438\u0444\u0442 \u0434\u0456\u0430\u0433\u0440\u0430\u043C\u0438
+fontDefinition.org.jkiss.dbeaver.erd.diagram.font.description = \u0428\u0440\u0438\u0444\u0442, \u044F\u043A\u0438\u0439 \u0432\u0438\u043A\u043E\u0440\u0438\u0441\u0442\u043E\u0432\u0443\u0454\u0442\u044C\u0441\u044F \u0434\u043B\u044F \u043D\u0430\u0439\u043C\u0435\u043D\u0443\u0432\u0430\u043D\u044C \u0441\u0443\u0442\u043D\u043E\u0441\u0442\u0435\u0439 \u0442\u0430 \u0430\u0442\u0440\u0438\u0431\u0443\u0442\u0456\u0432 \u043D\u0430 \u0434\u0456\u0430\u0433\u0440\u0430\u043C\u0456.
+fontDefinition.org.jkiss.dbeaver.erd.diagram.font.notation.label = \u0428\u0440\u0438\u0444\u0442 \u043F\u043E\u0437\u043D\u0430\u0447\u0435\u043D\u043D\u044F ER \u0434\u0456\u0430\u0433\u0440\u0430\u043C\u0438
+fontDefinition.org.jkiss.dbeaver.erd.diagram.font.notation.label = \u0428\u0440\u0438\u0444\u0442, \u044F\u043A\u0438\u0439 \u0432\u0438\u043A\u043E\u0440\u0438\u0441\u0442\u043E\u0432\u0443\u0454\u0442\u044C\u0441\u044F \u0434\u043B\u044F \u043F\u043E\u0437\u043D\u0430\u0447\u0435\u043D\u044C \u043D\u0430 ER \u0434\u0456\u0430\u0433\u0440\u0430\u043C\u0456.
```

Comment:
This property key is duplicated, should be description instead of label.
```suggestion
fontDefinition.org.jkiss.dbeaver.erd.diagram.font.notation.description= \u0428\u0440\u0438\u0444\u0442, \u044F\u043A\u0438\u0439 \u0432\u0438\u043A\u043E\u0440\u0438\u0441\u0442\u043E\u0432\u0443\u0454\u0442\u044C\u0441\u044F \u0434\u043B\u044F \u043F\u043E\u0437\u043D\u0430\u0447\u0435\u043D\u044C \u043D\u0430 ER \u0434\u0456\u0430\u0433\u0440\u0430\u043C\u0456.
```

Reply:
![image](https://github.com/dbeaver/dbeaver/assets/61122611/a21a9c70-50ec-4b1e-a224-d6ef01d69303)
Hello
I understand that your proposal is logical and should be accepted. But note that the keys are also duplicated in the original 😉

Reply:
Yeh, thanks, that's true. We will fix it.